zap/src/main/java/org/zaproxy/zap/extension/alert/AlertAPI.java

```diff
@@ -427,6 +428,7 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
             newAlert.setCweId(cweId);
             newAlert.setWascId(wascId);
             extension.alertFound(newAlert, msg.getHistoryRef());
+            Stats.incCounter("stats.ui.alert.api.add");
```

Comment:
I was going to comment about this having `ui` and checked the stats page to know of a more appropriate key, we already know how many by looking at `stats.api.call`, sorry.

Reply:
Oh okay, I'll roll this back.